packages/core/src/dialects/mariadb/index.ts

```diff
@@ -107,7 +107,7 @@ export class MariaDbDialect extends AbstractDialect {
   }
 
   escapeString(value: string) {
-    return escapeMysqlString(value);
+    return escapeMariaDbString(value);
```

Comment:
considering it's the same function, I wouldn't duplicate the code. Maybe rename the function to `escapeMysqlMariaDbString` instead?

Reply:
Done